pkg/cafs/writer.go

```diff
@@ -137,7 +137,7 @@ func pFlush(
 			err = destination.Put(context.TODO(), pather(leafKey.String()), bytes.NewReader(buffer), storage.OverWrite)
 		}
 		if err != nil {
-			errC <- fmt.Errorf("write segment file: %v", err)
+			errC <- fmt.Errorf("write segment file: %s, err: %v", pather(leafKey.String()), err)
```

Comment:
another way to put it using error stacking is: 
```suggestion
			errC <- fmt.Errorf("write segment file: %s, err: %w", pather(leafKey.String()), err)
```

Reply:
done